### PR TITLE
Update data-move-faq.md

### PR DESCRIPTION
**Pull request by andybergen**
Removed the "you will move no matter what" Q/A.  Cleaned up some other questions.  Replaced O365 with M365.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/data-move-faq.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Here are answers to general questions about moving core data to a new datacenter
   
 ## What customers are eligible to request a move?
   
-Existing Office 365 commercial customers who selected a country eligible for the new datacenter geo will be able to request a move.  The program exists only for tenants with an eligible country code assigned to the Office 365 tenant to migrate core customer data at rest for eligible workloads to the corresponding Office 365 datacenter geo.  Please refer to the [How to request your data move](request-your-data-move.md) page to confirm country eligibility.   
+Existing Microsoft 365 commercial customers who selected a country eligible for the new datacenter geo will be able to request a move.  The program exists only for tenants with an eligible country code assigned to the Microsoft 365 tenant to migrate core customer data at rest for eligible workloads to the corresponding Microsoft 365 datacenter geo.  Please refer to the [How to request your data move](request-your-data-move.md) page to confirm country eligibility.
```
Comment (JohanFreelancer9) [inline Enterprise/data-move-faq.md:25]:
```suggestion
Existing Microsoft 365 commercial customers who selected a country eligible for the new datacenter geo will be able to request a move. The program exists only for tenants with an eligible country code assigned to the Microsoft 365 tenant to migrate core customer data at rest for eligible workloads to the corresponding Microsoft 365 datacenter geo. Please refer to the [How to request your data move](request-your-data-move.md) page to confirm country eligibility.   
```

File: Enterprise/data-move-faq.md [hunk 2]
```diff
@@ -63,12 +53,12 @@ Teams files are stored in SharePoint Online and Teams chat files are stored in O
   
 ## What is the impact of having different services located in different geos?
 
-Some of the Office 365 services may be located in different geos for some existing customers and for customers that are in the middle of the move process.  Our services run independently of each other and there is no impact to the user experience if this is the case.  However, for data residency purposes, a tenant migration cannot be considered as complete until both Exchange Online and SharePoint 
+Some of the Microsoft 365 services may be located in different geos for some existing customers and for customers that are in the middle of the move process.  Our services run independently of each other and there is no impact to the user experience if this is the case.  However, for data residency purposes, a tenant migration cannot be considered as complete until both Exchange Online and SharePoint
```
Comment (JohanFreelancer9) [inline Enterprise/data-move-faq.md:56]:
```suggestion
Some of the Microsoft 365 services may be located in different geos for some existing customers and for customers that are in the middle of the move process. Our services run independently of each other and there is no impact on the user experience if this is the case. However, for data residency purposes, a tenant migration cannot be considered as complete until both Exchange Online and SharePoint 
```

File: Enterprise/data-move-faq.md [hunk 3]
```diff
@@ -63,12 +53,12 @@ Teams files are stored in SharePoint Online and Teams chat files are stored in O
...
+Some of the Microsoft 365 services may be located in different geos for some existing customers and for customers that are in the middle of the move process.  Our services run independently of each other and there is no impact to the user experience if this is the case.  However, for data residency purposes, a tenant migration cannot be considered as complete until both Exchange Online and SharePoint 
 Online/OneDrive for Business are migrated to the same datacenter geo.
   
-## Will new Office 365 customers be automatically provisioned in the new datacenter geos?
+## Will new Microsoft 365 customers be automatically provisioned in the new datacenter geos?
   
-Yes. Once a new datacenter geo is available, new Office 365 for Business customers who select a country eligible for the new geo as their country during sign-up will have their core customer data stored at rest in the new datacenter geo.
+Yes. Once a new datacenter geo is available, new Microsoft 365 who select a country eligible for the new geo as their country during sign-up will have their core customer data stored at rest in the new datacenter geo.
```
Comment (JohanFreelancer9) [inline Enterprise/data-move-faq.md:61]:
```suggestion
Yes. Once a new datacenter geo is available, new Microsoft 365 customers who select a country eligible for the new geo as their country during sign-up will have their core customer data stored at rest in the new datacenter geo.
```

File: Enterprise/data-move-faq.md [hunk 4]
```diff
@@ -63,12 +53,12 @@ Teams files are stored in SharePoint Online and Teams chat files are stored in O
...
   
  ## Where is my core customer data is located?
```
Comment (JohanFreelancer9) [inline Enterprise/data-move-faq.md:63]:
```suggestion
 ## Where is my core customer data located?
```

File: Enterprise/data-move-faq.md [hunk 5]
```diff
@@ -90,23 +80,20 @@ It is not possible for us to remove you from the process after you submit your r
...
 ## What happens if I do not request a move before the deadline?
   
- We may be able to accept request on an exception basis to grant your tenant a committed deadline to complete the move.  Please  
-contact [Office 365 Support](https://go.microsoft.com/fwlink/p/?LinkID=522459) to make the request.  Recall that some workloads may move to your new geo even without an opt-in request 
-as taking no action results in Microsoft being able to move your core customer data at rest to your new datacenter geo over time as 
-part of service management and optimization.  Your data can only potentially move to your new datacenter geo, not to any other geo.  
-We notify via Message Center when such a service management move has completed.
-  
- ## What if I want to move my data in order to get better network performance?
+We may be able to accept request on an exception basis to grant your tenant a committed deadline to complete the move.  Please
```
Comment (JohanFreelancer9) [inline Enterprise/data-move-faq.md:83]:
```suggestion
We may be able to accept requests on an exception basis to grant your tenant a committed deadline to complete the move. Please  
```

File: Enterprise/data-move-faq.md [hunk 6]
```diff
@@ -90,23 +80,20 @@ It is not possible for us to remove you from the process after you submit your r
...
-  
- ## What if I want to move my data in order to get better network performance?
+We may be able to accept request on an exception basis to grant your tenant a committed deadline to complete the move.  Please  
+contact [Office 365 Support](https://go.microsoft.com/fwlink/p/?LinkID=522459) to make the request.
+
+## What if I want to move my data in order to get better network performance?
   
-Being close to an Office 365 datacenter is not a guarantee for a better networking performance. There are many factors and components that impact the network performance between the end user and the Office 365 service. For more information about this and performance tuning see [Network planning and performance tuning for Office 365](network-planning-and-performance.md).
+Physical proximity to a Microsoft 365 datacenter is not a guarantee for a better networking performance. There are many factors and components that impact the network performance between the end user and the Microsoft 365 service. For more information about this and performance tuning see [Network planning and performance tuning for Office 365](network-planning-and-performance.md).
```
Comment (JohanFreelancer9) [inline Enterprise/data-move-faq.md:88]:
```suggestion
Physical proximity to a Microsoft 365 datacenter is not a guarantee for a better networking performance. There are many factors and components that impact the network performance between the end-user and the Microsoft 365 service. For more information about this and performance tuning see [Network planning and performance tuning for Office 365](network-planning-and-performance.md).
```

File: Enterprise/data-move-faq.md [hunk 7]
```diff
@@ -120,11 +107,11 @@ See [During and after your data move](during-and-after-your-data-move.md) for a
   
 Watch the Office 365 Message Center for confirmation that the move of each service's data is complete. When each service's data is moved, we'll post a completion notice so you'll get three completion notices: one each for Exchange Online, SharePoint Online, and Skype for Business Online.  You can also verify the location of your customer data at rest via the Data Location section under your Organization Profile in the Microsoft 365 Admin Center.  
   
-## I am an Office 365 customer in one of the new datacenter geos, but when I signed up, I selected a different country. How can I be moved to the new datacenter geo?
+## I am an Microsoft 365 customer in one of the new datacenter geos, but when I signed up, I selected a different country. How can I be moved to the new datacenter geo?
```
Comment (JohanFreelancer9) [inline Enterprise/data-move-faq.md:110]:
```suggestion
## I am a Microsoft 365 customer in one of the new datacenter geos, but when I signed up, I selected a different country. How can I be moved to the new datacenter geo?
```

File: Enterprise/data-move-faq.md [hunk 8]
```diff
@@ -138,37 +125,31 @@ Yes, however the process will not be as seamless as if Microsoft were to perform
...
-
-Yes.
+## Will Microsoft 365 tenants hosted in the new datacenters be available to users outside of the country?
   
-## Will Office 365 tenants hosted in the new datacenters be available to users outside of the country?
-  
-A. Yes. Microsoft maintains a large global network with public Internet connections in more than 130 locations in 35 countries around the world with peering agreements with more than 2,700 Internet Service Providers (ISPs). Users will be able to access the datacenters from wherever they are on the Internet.
+Yes. Microsoft maintains a large global network with public Internet connections in more than 130 locations in 35 countries around the world with peering agreements with more than 2,700 Internet Service Providers (ISPs). Users will be able to access the datacenters from wherever they are on the Internet.
+
+## My tenant has configured the [Multi Geo add-on](https://aka.ms/multi-geo).  Can I still enroll in my tenant in the Microsoft 365 Move Program to change my default geo and move any user not in a satellite region to the new default geo?
```
Comment (JohanFreelancer9) [inline Enterprise/data-move-faq.md:136]:
```suggestion
## My tenant has configured the [Multi Geo add-on](https://aka.ms/multi-geo). Can I still enroll in my tenant in the Microsoft 365 Move Program to change my default geo and move any user not in a satellite region to the new default geo?
```

File: Enterprise/data-move-faq.md [hunk 9]
```diff
@@ -138,37 +125,31 @@ Yes, however the process will not be as seamless as if Microsoft were to perform
...
 
-## My tenant is configured for [Office 365 Multi Geo](https://aka.ms/multi-geo).  Can I still enroll in my tenant in the Office 365 Move Program to change my default geo and move any user not in a satellite region to the new default geo?
+Yes, your tenant is eligible to enroll.  We will move all EXO mailboxes from your current default geo to your new local datacenter geo.  We will not move any EXO mailboxes configured in Multi Geo satellite regions to continue to respect satellite region data residency as you’ve intended.
```
Comment (JohanFreelancer9) [inline Enterprise/data-move-faq.md:138]:
```suggestion
Yes, your tenant is eligible to enroll. We will move all EXO mailboxes from your current default geo to your new local datacenter geo.  We will not move any EXO mailboxes configured in Multi Geo satellite regions to continue to respect satellite region data residency as you’ve intended.  
```

File: Enterprise/data-move-faq.md (hunk 4 above)
Comment (JoeDavies-MSFT) [reply Enterprise/data-move-faq.md:63]: "new Microsoft 365 who" should be "new Microsoft 365 customers who"

**Pull request merged**